packages/amplication-server/src/core/queue/queue.module.ts

```diff
@@ -0,0 +1,35 @@
+import { Module } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { ClientsModule, Transport } from '@nestjs/microservices';
+import { QueueService, QUEUE_SERVICE_NAME } from './queue.service';
+
+export const BROKER_IP_ENV_KEY = 'BROKER_IP_ENV_KEY';
+
+@Module({
```

Comment:
why did you decide to go with '@Module' and not '@Client'?
https://docs.nestjs.com/microservices/kafka#client